packages/web-service/src/pages/common/yes-no-conditional.js

```diff
@@ -1,34 +1,36 @@
 import Joi from 'joi'
 import pageRoute from '../../routes/page-route.js'
 
-export const validator = async payload => {
+export const validator = async (payload, skipValidations) => {
   if (!payload['yes-no']) {
     Joi.assert(payload, Joi.object({
       'yes-no': Joi.any().required()
     }).options({ abortEarly: false, allowUnknown: true }))
   }
 
-  if (payload['yes-no'] === 'yes') {
+  if (payload['yes-no'] === 'yes' && (!skipValidations || (skipValidations && !skipValidations.includes('yes-conditional-input')))) {
     Joi.assert(payload, Joi.object({
       'yes-conditional-input': Joi.string().trim().required().replace('\r\n', '\n').max(4000)
     }).options({ abortEarly: false, allowUnknown: true }))
   }
 
-  if (payload['yes-no'] === 'no') {
+  if (payload['yes-no'] === 'no' && (!skipValidations || (skipValidations && !skipValidations.includes('no-conditional-input')))) {
```

Comment:
Again, we could use optional chaining here if we wanted:
```suggestion
  if (payload['yes-no'] === 'no' && !skipValidations?.includes('no-conditional-input')) {
```

Reply:
I'll do the same as above here